frontend/src/components/media_search.svelte

```diff
@@ -98,30 +96,11 @@
       </a>
     {/each}
   </div>
-  <div class="flex space-x-1 justify-center p-1">
-    {#if currentMediaAmount > mediaStartAmount}
-      <button
-        on:click={() => {
-          changeMediaAmount("loadless")
-        }}
-        id="loadless"
-        class="btn"
-      >
-        Show less
-      </button>
-    {/if}
-    {#if currentMediaAmount < meilisearch.nbHits}
-      <button
-        on:click={() => {
-          changeMediaAmount("loadmore")
-        }}
-        id="loadmore"
-        class="btn btn-primary"
-      >
-        Show more
-      </button>
-    {/if}
-  </div>
+  {#if meilisearch.hits.length === meilisearch.limit}
+    <InfiniteLoading on:infinite={loadMoreData} />
```

Comment:
Now that almost every page got an infinite loader we probably have to redesign the navbar, so the user is able to access the things that are only at the bottom - Especially on mobile

Reply:
#174